gate-core/src/main/java/com/netflix/spinnaker/gate/services/TaskService.java

```diff
@@ -22,11 +22,15 @@
 import java.util.*;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
 @Service
 public class TaskService {
 
+  @Value("${taskService.maxNumberOfPolls:32}")
```

Comment:
There's a trend away from `@Value` and towards `@ConfigurationProperties`.  I don't see an existing `@ConfigurationProperties`-annotated class for `taskService`.  If this is really the first property, it's the easiest time to add one.